FiveCalls/FiveCalls/CategorizedIssuesViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CategorizedIssuesViewModel.swift
+//  FiveCalls
+//
+//  Created by Ben Scheirman on 1/29/19.
+//  Copyright © 2019 5calls. All rights reserved.
+//
+
+import Foundation
+
+class CategorizedIssuesViewModel {
+    let category: Category
+    var issues: [Issue]
+
+    var name: String {
+        return category.name
+    }
+
+    init(category: Category, issues: [Issue]) {
+        self.category = category
+        self.issues = issues
+    }
+}
+
+extension CategorizedIssuesViewModel : Equatable, Hashable {
```

Comment:
I don't think you need to specify `Equatable` here as `Hashable` descends from it.